tools/cli/go-whisk-cli/commands/activation.go

```diff
@@ -114,7 +114,11 @@ var activationGetCmd = &cobra.Command{
     PreRunE: setupClientConfig,
     RunE: func(cmd *cobra.Command, args []string) error {
         var field string
+        var err error
 
+        if args, err = lastFlag(args); err != nil {
+          return err
```

Comment:
@mdeuser, do we want a debug message in each of these if statements stating an error occurred?